src/identity/Identity.ts

```diff
@@ -144,6 +371,30 @@ export default class Identity extends PublicIdentity {
     )
   }
 
+  /**
+   * @description Decrypts Asymmetrical using random secret key (UInt8Array)
+   * @param encrypted An excrypted message/data
+   * @param boxPublicKey A random secret key
+   * @returns `Crypto.decryptAsymmetric(...)`
```

Comment:
Let's document what this actually returns, aka what `Crypto.decryptAsymmetric()` returns.